muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/me/MeRESTService.java

```diff
@@ -72,10 +72,28 @@ public class MeRESTService {
 
   @Inject
   private UserGroupGuidanceController userGroupGuidanceController;
-  
+
   @Inject
   private ChatController chatController;
   
+  /**
+   * Returns the server side locale for current user or default if not logged in.
+   * 
+   * Returns:
+   * {
+   *    lang: "en"
+   * }
+   */
+  @GET
+  @Path("/locale")
+  @RESTPermit (handling = Handling.UNSECURED)
+  public Response getLocale() {
+    Locale locale = localSessionController.getLocale();
+    String lang = (locale == null || locale.getLanguage() == null) ? "FI" : locale.getLanguage().toUpperCase();
```

Comment:
Shouldn't this be lowercase to indicate language? After all, only country names are in uppercase (e.g. `en-GB` or `en-US`)